tests/z85.rs

```diff
@@ -1,8 +1,8 @@
-use quickcheck::{quickcheck, Arbitrary, Gen};
-use rand::Rng;
-use zmq::{z85_decode, z85_encode, DecodeError, EncodeError};
+//use quickcheck::{quickcheck, Arbitrary, Gen};
+//use rand::Rng;
+//use std::iter;
```

Comment:
If this code is fully redundant I think it'd be better to fully remove it from the codebase. Same goes for the comments later in this file.

Reply:
I think it is still a valid test, but one that just needs to be updated eventually so I'll opt to leave it.